modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/rules/editor/Conditions.es.js

```diff
@@ -72,6 +72,7 @@ function FieldOperator({
 		<>
 			<Timeline.FormGroupItem>
 				<FieldStateless
+					id="field-operator-id-test"
```

Comment:
since this component is not mocked and is the main component used, makes sense to have "test" on the class? wouldn't be better to just be `field-operator-id`? i don't have the context so sorry if i misunderstood!